.circleci/config.yml

```diff
@@ -0,0 +1,232 @@
+    # https://circleci.com/blog/how-to-handle-java-oom-errors/
+    # Gradle itself doesn't usually reach 1G, so as long as we can keep tasks under 3G, we should
+    # have room in Circle's 4G to get our work done.
+    JAVA_TOOL_OPTIONS: -Xmx3g
+    TERM: dumb
+
+jobs:
+  api-local-test:
+    docker:
+      - image: allofustest/workbench:buildimage-0.0.12
+      - image: mysql:5.7
+        environment:
+          - MYSQL_ROOT_PASSWORD=ubuntu
+          - MYSQL_USER=ubuntu
+          - MYSQL_PASSWORD=ubuntu
+    working_directory: ~/data-browser
+    environment:
+      JAVA_TOOL_OPTIONS: -Xmx3g
+      TERM: dumb
+      MYSQL_ROOT_PASSWORD: ubuntu
+    steps:
+      - checkout
+      - run:
+          name: Fetch Submodules
+          command: git submodule update --init --recursive
+      - restore_cache:
+          keys:
+          - api-cache-{{ checksum "~/data-browser/public-api/build.gradle" }}
+          - api-cache-
+      - run:
+          working_directory: ~/data-browser
+          command: ci/activate_creds.sh api/circle-sa-key.json
+      - run:
+          # MySQL sometimes refuses connections by the time we attempt to apply
+          # data migrations. Watch the port for 2m for startup.
+          name: Await MySQL startup
+          command: dockerize -wait tcp://127.0.0.1:3306 -timeout 2m
+      - run:
+          working_directory: ~/data-browser/public-api
+          command: ./project.rb start-local-public-api && ./project.rb run-local-public-api-tests && ./project.rb stop-local-public-api
+      - save_cache:
+          paths:
+            - ~/.gradle
+            - ~/.m2
+            - ~/data-browser/public-api/build/exploded-api/WEB-INF/lib/
+          key: api-cache-{{ checksum "~/data-browser/public-api/build.gradle" }}
+
+  api-deploy-to-test:
+    <<: *java_defaults
+    steps:
+      - checkout
+      - run:
+          command: git submodule update --init --recursive
+      # Note: most of the time spent here appears to be in Gradle / App Engine
+      # deployment. We tried more aggressively caching outputs via Cirlce
+      # workspaces, but that seemed to have a negligible effect on speed. It's
+      # also tricky to pick specific sub directories since the API deploy
+      # touches several top level folders {public,common,}api.
+
+      # TODO: Fix the deploy script to work on the correct new project.
+      - restore_cache:
+          keys:
+          - api-cache-{{ checksum "~/data-browser/api/build.gradle" }}
+          - api-cache-
+      - run:
+          working_directory: ~/data-browser
+          command: ci/activate_creds.sh api/circle-sa-key.json
+      - deploy:
+          name: Deploy to App Engine
+          working_directory: ~/data-browser/api
+          command: |
+            ./project.rb deploy \
+              --project all-of-us-workbench-test \
+              --account circle-deploy-account@all-of-us-workbench-test.iam.gserviceaccount.com \
```

Comment:
Could you create a circle-deploy-account under `aou-db-test`? Just added your pmi-ops as an owner